generators/app/templates/package.json

```diff
@@ -26,7 +26,8 @@
   },
   "peerDependencies": {
     "react": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.x"
```

Comment:
React could only be a peer-dependency.  If it is a dependency, it will be installed nested inside the package. 

It is the same as this issue here:
https://github.com/Hacker0x01/react-datepicker/issues/122

This leads to dependencies being added to the bundle more than once. 

We should follow what other libraries are doing:
https://github.com/cloudflare/cf-ui/blob/master/packages/cf-builder-table/package.json#L20
https://github.com/mineral-ui/mineral-ui/blob/master/packages/archetype/package.json#L21-L23
https://github.com/palantir/blueprint/blob/master/packages/core/package.json#L19-L23